.github/workflows/upstream_sync.yml

```diff
@@ -0,0 +1,46 @@
+name: 'Upstream Sync'
+
+on:
+  schedule:
+    - cron:  '0 7 * * *'
+    # every day at 7am
+
+  workflow_dispatch:  # click the button on Github repo!
+
+jobs:
+  sync_latest_from_upstream:
+    runs-on: ubuntu-latest
+    name: Sync latest commits from upstream repo
+
+    steps:
+    - name: Checkout target repo
+      uses: actions/checkout@v3
+      with:
+        ref: main
+        # REQUIRED if your upstream repo is private (see wiki)
+        persist-credentials: false
+
+    - name: Sync upstream changes
+      id: sync
+      uses: aormsby/Fork-Sync-With-Upstream-action@v3.4
+      with:
+        target_sync_branch: main
+        # REQUIRED 'target_repo_token' exactly like this! This token is created automatically for each workflow run: https://docs.github.com/en/actions/security-guides/automatic-token-authentication
+        target_repo_token: ${{ secrets.GITHUB_TOKEN }}
+        upstream_sync_branch: main
+        upstream_sync_repo: aeharding/voyager
+
+        # Set test_mode true to run tests instead of the true action!!
+        test_mode: false
+      
+    # Step 3: Display a sample message based on the sync output var 'has_new_commits'
+    - name: New commits found
+      if: steps.sync.outputs.has_new_commits == 'true'
+      run: echo "New commits were found to sync."
+    
+    - name: No new commits
+      if: steps.sync.outputs.has_new_commits == 'false'
+      run: echo "There were no new commits."
```

Comment:
Interesting, I could use this logic for my other forks:

https://github.com/sharunkumar/Summer2024-Internships

https://github.com/sharunkumar/New-Grad-Positions

Reply:
I don't think it's a GitHub workflows-wide output, rather an output variable specific to the `Fork-Sync-With-Upstream-action` action:

https://github.com/aormsby/Fork-Sync-With-Upstream-action/blob/f0889e09db4cb5657459c9256a9fb62a740d4f93/run/get_updates.sh#L49-L51

Reply:
Yeah I got that, just saying that it could be helpful in those repos